calculating_n50_assemblies.py

```diff
@@ -0,0 +1,91 @@
+import glob
+import re
+import gzip
+import operator
+import sys
+import os
+
+def main(in_dir = None, out_dir = None):
+    if os.path.exists(in_dir) == False:
+        sys.exit("Input directory or path incorrect. Please check")
+
+    if os.path.exists(out_dir) == False:
+        os.makedirs(out_dir)
+    
+    file1 = open(os.path.join(out_dir+'/summary_concise.txt'),'w')
```

Comment:
This is the wrong usage for os.path.join, please refer to https://docs.python.org/3/library/os.path.html#os.path.join

Reply:
This applies to all of your usages of os.path.join

Reply:
For line 12 and 13, I mean if the output directory does not exist (like you want to write output files to a folder that you didnt create), then make one.

Reply:
line 15 is what the comment is on, github just shows the preceding 3 lines in addition.

Reply:
changed to os.mkdir . Need to create only a single directory so previous was a mistake. 
os.path.join arguments separated by comma:
open(os.path.join(out_dir,'summary_concise.txt'),'w')